modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,33 @@
+"""
+Modular Exponential
+"""
+
+def mod_exponent(base, power, mod):
+    """
```

Comment:
But here there is no such description provided and followed the guidelines as told by @singhpratyush